src/tutorial/microbe_stage/DayNightTutorial.cs

```diff
@@ -23,6 +30,8 @@ public override bool CheckEvent(TutorialState overallState, TutorialEventType ev
             {
                 if (!HasBeenShown && CanTrigger && !overallState.TutorialActive())
                 {
+                    EnvironmentPanel.ShowPanel = true;
+                    HUDBottomBar.EnvironmentPressed = true;
```

Comment:
I think GUI state applying should more logically be in the ApplyGUIState method. Also I think it would be much safer to make a single method that ensures whatever the GUI wants to show is visible correctly, rather than trying to set the panel visibility and related button here.